klsreader/protocol/controllerdata.py

```diff
@@ -40,7 +40,7 @@ def _unpackPacketA(self, data):
         (self.throttle, self.brakePedal, self.brakeSwitch, self.footSwitch,
         self.forwardSwitch, self.reverse, self.hallA, self.hallB, self.hallC,
         self.batteryVoltage, self.motorTemp, self.controllerTemp, self.settingDir,
-        self.actualDir) = unpack('2x5B4?3B2?3x', data)
+        self.actualDir) = unpack('!2x5B4?3B2?3x', data)
```

Comment:
See if this breaks anything and if so reject the PR and I can fix it, or you can recommit over this.